main.cpp

```diff
@@ -80,43 +96,95 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+bool soundPlaying(double volume, int onSwitch=0)
+{
+    ignoreUnused(volume, onSwitch);
+    return {};
+}
 /*
  2)
  */
-
+void getWeather(double temp, bool detectRain, char climateZone)
+{
+    ignoreUnused(temp, detectRain, climateZone);
+
+}
 /*
  3)
  */
-
+void keyboardType(int keys = 88, bool electric = true)
+{
+    ignoreUnused(keys, electric);
+
+}

```

Comment:
Get in the habit of removing empty lines
they are unnecessary and make your code look sloppy
- [ ] revise